mqemitter.js

```diff
@@ -92,6 +95,9 @@ MQEmitter.prototype.emit = function emit (message, cb) {
   if (this.concurrency > 0 && this.current >= this.concurrency) {
     this._messageQueue.push(message)
     this._messageCallbacks.push(cb)
+    if (!this.doing) {
+      this._released()
```

Comment:
Please add a [`process.emitWarning()`](https://nodejs.org/api/process.html#process_process_emitwarning_warning_options) here. This _should not happen_ and it's either a bug in aedes or the client and we should not be relying on a "hot fix" for things to work properly.